skit.Core/Companies/Entities/Company.cs

```diff
@@ -8,16 +8,33 @@ namespace skit.Core.Companies.Entities;
 
 public sealed class Company : Entity
 {
-    public string Name { get; set; }
-    public string? Description { get; set; }
-    public CompanySize Size { get; set; }
-    public string? Links { get; set; }
-    
-    public Guid OwnerId { get; set; }
+    public string Name { get; private set; }
+    public string? Description { get; private set; }
+    public CompanySize Size { get; private set; }
+    public string? Links { get; private set; }
+
+    public Guid OwnerId { get; private set; }
     public User Owner { get; set; }
+
+    public List<Address> Addresses { get; }
```

Comment:
to spróbujemy zrobić tak ja pokazywał Kacper.
<img width="395" alt="Zrzut ekranu 2023-11-08 124127" src="https://github.com/skni-kod/skitBackend/assets/80985555/3eb92772-06dc-4804-9caa-aba6d794b734">